inspirehep/modules/records/jsonschemas/records/elements/id.json

```diff
@@ -0,0 +1,155 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "anyOf": [
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE ID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "INSPIRE-\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE BAI"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "(\\w+\\.)+\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "ORCID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]",
```

Comment:
No it won't, but it's a harmless addition. I believe we only have capital X, though: https://inspirehep.net/search?ln=en&ln=en&p=100__j%3A%2F%5Cd%7B3%7Dx%2F&of=text&ot=100__j&rg=350.